debug_toolbar/panels/templates/panel.py

```diff
@@ -85,7 +85,8 @@ def _store_template_info(self, sender, **kwargs):
 
         # Skip templates that we are generating through the debug toolbar.
         if (isinstance(template.name, six.string_types) and
-                template.name.startswith('debug_toolbar/')):
```

Comment:
I think I agree with @kezabelle . It's probably going to cause fewer headaches for people if we keep the 'debug_toolbar/' skip hard-coded instead of requiring users who customize the list to include it.

On the other hand, if it makes developing and debugging the toolbar itself easier, I don't think it's a big deal, as long as the default value is clearly documented.